client/my-sites/checkout/composite-checkout/hooks/use-create-payment-complete-callback.tsx

```diff
@@ -237,6 +238,13 @@ export default function useCreatePaymentCompleteCallback( {
 				return;
 			}
 
+			// We need to do a hard redirect if we're redirecting to the stepper.
+			// Since stepper is self-contained, it doesn't load properly if we do a normal history state change
+			if ( isURL( url ) || url.startsWith( '/setup' ) ) {
+				window.location.href = url;
```

Comment:
We must use the pending page, but I think that if you change this to the following, it will work:

```
				absoluteRedirectThroughPending( url, {
					siteSlug,
					orderId: transactionResult.order_id,
					receiptId: transactionResult.receipt_id,
				} );
```

Reply:
Ah, I see what you mean. I think changing to an absolute url makes the most sense. I'll try that.